app/views/admin_posts/show.html.erb

```diff
@@ -6,16 +6,20 @@
     <!--/descriptions-->
     <!--subnav-->
     <div class="navigation actions module" role="navigation">
-      <% if policy(@admin_post).can_post? %>
+      <% if logged_in_as_admin? %>
```

Comment:
Could we update [this test](https://github.com/otwcode/otwarchive/blob/2055928e5d26ae0271fe121aa3f32f0c3c0d0a8e/features/admins/admin_post_news.feature#L275) to use a role that isn't able to access the edit and delete links but should be able to access the unreviewed comment link?